NEWS.md

```diff
@@ -1,3 +1,7 @@
+# naomi.utils 0.0.13
+
+* Copy `get_age_groups()` from naomi package
```

Comment:
Add a bit more about the rationale for this (what you put in the commit message).